app/frontend/src/search/ui/pagination.test.js

```diff
@@ -0,0 +1,7 @@
+import { renderPagination } from './pagination';
+
+describe('renderPagination', () => {
+  test('returns a function', () => {
+    expect(typeof renderPagination).toBe('function');
+  });
+});
```

Comment:
tbh i would remove this test file completely, it a very lame test (even tho im perfectly aware i created it first!)

Reply:
I agree and I'm happy with that